src/cli/Cli.ts

```diff
@@ -176,9 +174,28 @@ export class Cli {
       return this.closeWithError(validationResult, optionsWithoutShorts, true);
     }
 
+    const end = process.hrtime.bigint();
+    timings.core.push(Number(end - start));
+
     return Cli
       .executeCommand(this.commandToExecute.command, optionsWithoutShorts)
-      .then(_ => process.exit(0), err => this.closeWithError(err, optionsWithoutShorts));
+      .then(_ => {
```

Comment:
Maybe nice to refactor this to Async/await right away...

Reply:
I didn't want to include any changes unrelated to the PR 😊 There are more places that use promises so I suggest that we hold off with this change and take it separately